### PR TITLE
Makes Password door sounds not hardcoded.

### PR DESCRIPTION
**Pull request by Potato-Masher**
## About The Pull Request
Changes the sounds on password locked doors to not be hardcoded; now being controlled by variables. 

## Why It's Good For The Game
Allows admins (me) to edit the sounds for events and event maps. Probably some benefit to mappers who come along this type of door as well.

## Changelog

None as this isn't player facing.

### CODE REVIEW

File: code/game/machinery/doors/passworddoor.dm [hunk 1]
```diff
@@ -12,6 +12,9 @@
 	var/password = "Swordfish"
 	var/interaction_activated = TRUE //use the door to enter the password
 	var/voice_activated = FALSE //Say the password nearby to open the door.
+	var/doorOpen = 'sound/machines/blastdoor.ogg' //Sound used upon opening.
+	var/doorClose = 'sound/machines/blastdoor.ogg' //Sound used upon closing.
+	var/doorDeny = 'sound/machines/buzz-sigh.ogg' //Sound used upon denying.
```
Comment (dragomagol) [inline code/game/machinery/doors/passworddoor.dm:17]: Should be in snake_case

Comment (Potato-Masher) [reply code/game/machinery/doors/passworddoor.dm:17]: Curious... airlock.dm (used as a reference) had camelCase everywhere...

**Comment by Potato-Masher** [comment]
Ain't no way those failed checks are my fault

**Comment by Potato-Masher** [comment]
I know not why the tests fail now

**Comment by dragomagol** [comment]
image

I see the problem but I'm not sure why it's failing. Maybe try mapmerging from master?

**Pull request merged**